task/rhacs-deployment-check/3.71/README.md

```diff
@@ -0,0 +1,72 @@
+# Red Hat Advanced Cluster Security Deployment Check Task
+
+Check a deployment manifest against RHACS deploy lifecycle policies to validate a pipeline run using `roxctl`.
+
+## Prerequisites
+
+This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet).  It also requires configuration of secrets for the Central endpoint and an API token with at least CI privileges.
+
+<https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml
```

Comment:
instead of using raw.githubusercontent URL can we use the endpoint which tekton hub would be generating once this Task is merged? Format of the endpoint that Hub is going to generate would be
```
https://api.hub.tekton.dev/v1/resource/tekton/task/{TaskName}/{TaskVersion}/raw
```
so in your case the endpoint generated would be
```
https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-deployment-check/3.71/raw
```

Reply:
thanks for that. done.